demos/browser/ChimeMeetingsWrapper.js

```diff
@@ -0,0 +1,52 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+const AWS = require('aws-sdk');
+
+module.exports = class ChimeMeetingsWrapper {
+
+    // Create an AWS SDK Chime object. Region 'us-east-1' is currently required.
+    // Use the MediaRegion property below in CreateMeeting to select the region
+    // the meeting is hosted in.
+    
+
+    constructor(currentRegion, controlRegion, endpoint){
```

Comment:
Can you clarify the functional difference between `currentRegion` and `controlRegion`?

Reply:
Current region is hardcorded to us-east-1 in either handler or server.js and controlRegion is being passed as url params. Also in the case where controlRegion is not passed in we set controlRegion = currentRegion

Reply:
Why is `currentRegion` necessary though?

Reply:
updated to remove current region